src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier randFigure = new FigureSupplier();
+        randFigure.showFigure();
+
```

Comment:
```suggestion
        FigureSupplier supplier = new FigureSupplier();
        Figure[] randomFigures = new Figure[randomNumber];
        for (int i = 0; i < randomFigures.length; i++) {
            randomFigures[i] = supplier.getRandomFigure;
        }

```
аналогічно всі фіугри потрібно намалювати через виклик методу draw() в циклі